api/v1alpha1/README.md

```diff
@@ -4,24 +4,46 @@ This document is intended to provide context to the API definition.
 
 ## Device.Status
 
-### Conditions
-These are the core conditions that cover the ability of the device to accept/manage workloads.
+### Device.Status.Applications
 
 | Name    | Desc |
 | -------- | ------- |
-| Ready  | True if the Device is ready and can accept new containers otherwise False. This is also the only condition which can be set by the server, otherwise False. |
-| DiskPressure | True if pressure exists on the disk size—that is, if the disk capacity is low; otherwise False. |
-| PIDPressure | True if pressure exists on the processes—that is, if there are too many processes on the node, otherwise False. | 
-| MemoryPressure |True if pressure exists on the device memory—that is, if the device memory is low, otherwise False. |
-| CPUPressure |True if device is experiencing high CPU utilization that might affect the performance of the workloads running on that device, otherwise False. |
-| SystemdUnitsRunning | True is all specified systemd units are running, otherwise False. |
-| ContainersRunning | True is all specified containers are running, otherwise False. |
+| ID | Unique identifier for the application |
+| Name | A human readable name for the application |
```

Comment:
Would the name not be unique and able to serve as identifier like in K8s? Or better asked:
What would be examples of ID and name and where would they come from? Let's say the user deploys a Helm chart with a certain release name, would the application's name be the release name? The name of a deployment?

Reply:
ID could be internal and not exposed I don't see a problem with that.